client/scripts/controllers/server/notifications.ts

```diff
@@ -142,10 +153,12 @@ class NotificationsController {
   public update(n: Notification) {
     if (!this._store.getById(n.id)) {
       this._store.add(n);
+      m.redraw();
```

Comment:
This makes sense to me, but wondering if there's any way we can redraw once per message rather than on every event in the message?

Reply:
Every message is an event so not really. If redrawing too often is an issue we can refresh every X min or X events but I think mithril redraw algo is optimized enough that the redraws are not very costly.